src/navbar/menu.tsx

```diff
@@ -64,7 +64,7 @@ export const Menu = <T,>({
                         {...options}
                     >
                         {children}
-                        {selected && <LinkIndicator />}
+                        {isMobile && selected && <LinkIndicator />}
```

Comment:
actually I still don't see a case where this `LinkIndicator` is required. the dark blue border is already set on the `MobileWrapper`